lib/wallet/http.js

```diff
@@ -668,6 +668,36 @@ class HTTP extends Server {
       res.json(200, addr.toJSON(this.network));
     });
 
+   // Derive receive address
+    this.post('/wallet/:id/derivereceive', async (req, res) => {
```

Comment:
Why does this need to be POST, we are not modifying anything.
There are methods that are really similar:
`POST /wallet/:id/address` - create receive address
`POST /wallet/:id/change` - create change address
`POST /wallet/:id/nested` - create nested address

We could have these methods have similar API, like:
`GET /wallet/:id/address` - get receive address
`GET /wallet/:id/change` - get change address
`GET /wallet/:id/nested` - get nested address

If we limit these methods to depth (to be safe) as @tuxcanfly mentioned, would fill up other methods.